sql/moz-fx-data-shared-prod/telemetry/releases_latest/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Releases Latest
+description: |-
+  This is built on top of telemtry.releases and aims
+  to provide an entry on each day for release and beta channels
+  along with the newest firefox app version being available
```

Comment:
along with the latest Firefox version released on that day or earlier for those channels.